LayoutTests/fast/forms/ios/keyboard-stability-when-tapping-editable-element.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true ] -->
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+<script src="../../../resources/js-test.js"></script>
+<script src="../../../resources/ui-helper.js"></script>
+<style>
+body, html {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+}
+</style>
+<script>
+jsTestIsAsync = true;
+
+addEventListener("load", async () => {
+    description("This test verifies that switching first responder from a native text field to web content by tapping on an editable element does not cause the keyboard to quickly disappear and reappear. This test requires WebKitTestRunner.");
+
+    await UIHelper.setHardwareKeyboardAttached(false);
+
+    // Start by focusing a native text field.
+    await UIHelper.addChromeInputField();
+    await UIHelper.setTextInChromeInputField("Foo bar");
+    await UIHelper.selectChromeInputField();
+    await UIHelper.waitForKeyboardToShow();
+
+    keyboardWillHideCountBefore = await UIHelper.keyboardWillHideCount();
+
+    // Then tap to focus the editable body.
+    await UIHelper.activateElement(document.body);
+    await UIHelper.delayFor(100);
```

Comment:
Can we write the test without a `UIHelper.delayFor`?

What if we were to check `UIHelper.keyboardWillHideCount` after `UIHelper.waitForKeyboardToHide` and assert that the count only went up by one?

Reply:
Ah, that's a great idea — changed to the following:

```
…
    // Then tap to focus the editable body.
    await UIHelper.activateElement(document.body);

    document.body.blur();
    await UIHelper.waitForKeyboardToHide();

    keyboardWillHideCountAfter = await UIHelper.keyboardWillHideCount();
    shouldBe("keyboardWillHideCountAfter - keyboardWillHideCountBefore", "1");

    finishJSTest();
…
```

Thanks for the review!